addons/mail/static/src/widgets/form_renderer/form_renderer_tests.js

```diff
@@ -975,6 +975,75 @@ QUnit.test('chatter just contains "creating a new record" message during the cre
     );
 });
 
+QUnit.test('automatic loading of messages on scrolling of formview', async function (assert) {
+    assert.expect(3);
+
+    for (let i=0; i<50; i++) {
```

Comment:
Use const and apply liter

Reply:
@seb-odoo . not getting you actually.

Reply:
Forget it for const that was wrong.
But for linter: should have space around operators (=, <)

Reply:
Done